Prithvi011-021/016-ExportFunctionLib.js

```diff
@@ -0,0 +1,23 @@
+//Export the above functions as library (use module.exports). Write a function which takes
+// as input an arithmetic expression string example: “8 + 4 - 2 / 5 * 2”. 
+// Break the string on spaces to extract the numbers and arithmetic operations between them.
+//  Start solving it from left to right. For above expression the solution will be
+// a. 8 + 4 => 12
+// b. 12 - 2 => 10
+// c. 10 / 5 => 2
+// d. 2 * 2 => 4.
+// e. The final answer is 2. Ensure your solution is a recursive function.
+const arithmeticExp = require('./015-ImplementingCalculator')
```

Comment:
Modify this to be evaluated via a for loop so that the function can take an expression of any length